### DISCUSSION TITLE
Nuget Push doesn't understand the config changes about proxy on runtime with code

### DISCUSSION THREAD

**Discussion by slnkykrn** [NuGet/Home#11215] [Q&A]
Hello, 
I'm using nuget.protocol package for my CI/CD pipeline console application with .net 5. 
The application needs to get, download packages under a setted proxy setting. 
For pushing packages step I need to unset proxy , so I'm setting the proxy for get, download operations and then unsetting proxy on runtime.  But nuget push operation didn't understand the nuget.config file has been changed and so doesn't understand unsetting proxy operation. There isn't any problem when setting proxy and then get and download packages it works fine. But push operation just doesn't get it when .exe is running. 
So what is the reason of this situation and how can I force to reload settings when I pushing on runtime. 
Because I've just tried if I unset proxy and then exit console application and then start exe , pushing works. But on the runtime it just can't get the setting config (proxy configuration) has changed. 

 settings.Remove("config", new AddItem("HTTP_PROXY", "https://blac.blac"));
  settings.SaveToDisk();
-----------------------------
 await resource.Push(
                    packagePaths: packagePaths,
                    symbolSource: null,
                    timeoutInSecond: 2401
                    disableBuffering: false,
                    getApiKey: packageSource => null,
                    getSymbolApiKey: packageSource => null,
                    noServiceEndpoint: false,
                    skipDuplicate: true,
                    symbolPackageUpdateResource: null,
                    log: new NugetLogger(_loggerFactory)
                    );

Thank you.

**Comment by erdembayar** [comment]
Sorry for late response. Have you already resolve your issue?
We had regression and it was fixed https://github.com/NuGet/Home/issues/10902
If you're still facing this issue please give me dotnet sdk version you're using.
